vnext/ReactWindows.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27310.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29009.5
```

Comment:
>16.0.29009.5 [](start = 22, length = 12)

Has this any ramifications on the VS version needed to open the solution?

Reply:
No. The solution can actualy be opened by VS 2017. I chose to update the value to suggest that VS2019 is preferred.